net-p2p/deluge/deluge-1.3.12-r4.ebuild

```diff
@@ -44,14 +55,26 @@ RDEPEND=">=net-libs/rb_libtorrent-0.14.9[python]
 		gnome-base/librsvg
 		libnotify? ( dev-python/notify-python[${PYTHON_USEDEP}] )
 	)
-	setproctitle? ( dev-python/setproctitle[${PYTHON_USEDEP}] )
 	webinterface? ( dev-python/mako[${PYTHON_USEDEP}] )"
 
 python_prepare_all() {
-	local PATCHES=(
-		"${FILESDIR}"/${PN}-1.3.5-disable_libtorrent_internal_copy.patch
-		"${FILESDIR}"/${PN}-1.3.12-fix_scheduler_plugin.patch
+	local args=(
+		-e "/Compiling po file/a \\\tuptoDate = False"
 	)
+	sed -i "${args[@]}" -- 'setup.py' || die
+	args=(
+		-e 's|"new_release_check": True|"new_release_check": False|'
+		-e 's|"check_new_releases": True|"check_new_releases": False|'
+		-e 's|"show_new_releases": True|"show_new_releases": False|'
+	)
+	sed -i "${args[@]}" -- 'deluge/core/preferencesmanager.py' || die
+
+	loc_dir="${S}/deluge/i18n"
```

Comment:
In data sabato 23 aprile 2016 03:17:35 CEST, Jan Chren ha scritto:
> >  	)
> > 
> > +	sed -i "${args[@]}" -- 'setup.py' || die
> > +	args=(
> > +		-e 's|"new_release_check": True|"new_release_check": False|'
> > +		-e 's|"check_new_releases": True|"check_new_releases": False|'
> > +		-e 's|"show_new_releases": True|"show_new_releases": False|'
> > +	)
> > +	sed -i "${args[@]}" -- 'deluge/core/preferencesmanager.py' || die
> > +
> > +	loc_dir="${S}/deluge/i18n"
> 
> should be `local`

Do you mean "args" or "loc_dir"? "args" has already been declared local a few 
lines before.

-- 
Paolo Pedroni
paolo.pedroni<at>iol.it